admin/views/school/pupil-register.ejs

```diff
@@ -16,20 +16,28 @@
         <div>
             <h1 class="heading-xlarge">Pupil register</h1>
             <p class="lede">
+                During the trial pupils will take the check anonymously. Below is a list of 36 pre-populated pupil records.
+                <ul>
+                    <li><a href="https://irp0dn.axshare.com/prototype/login/irp0dn/#PupilReg=default&CSUM=1">As per Prototype</a></li>
+                    <li>Prototype Password: Tim3s</li>
+                </ul>
+                <!--
                 This list uses anonymous pupil data in the format Pupil 1, Pupil 2, etc. If you need to add more pupils please follow this pattern. Do not use real pupil data. To withdraw pupils, go to <a href="/school/pupils-not-taking-check">Pupils not taking the check</a>.
+                -->
             </p>
             <div class="button-wrapper">
                 <% if (isFeatureEnabled('pupilAdd')) { %>
                 <a class="button button-secondary" href="/school/pupil/add"> Add pupil </a>
                 <% } else { %>
                 <a class="button button-secondary disabled-link" href="#"> Add pupil </a>
                 <% } %>
-
+                <!--
```

Comment:
Does this need to be commented out?  Does't the feature toggling do enough?

Reply:
Comments can be removed now that it's been confirmed that feature toggling will work on this environment. I'll do the necessary changes.